.circleci/config.yml

```diff
@@ -31,11 +31,20 @@ jobs:
             sleep 10s &&
             docker-compose up -d
 
-      # TODO Uncomment when rubocop fixes are completed
-#      - run:
-#          name: "Rubocop"
-#          command: |
-#            docker-compose exec web bundle exec rubocop
+      - run:
+          name: "Rubocop"
+          command: |
+            docker-compose exec web bundle exec rubocop
+
+      - run:
+          name: "Rubocop-Rails"
+          command: |
+            docker-compose exec web bundle exec rubocop -R
+
+      - run:
+          name: "Rubocop-Rspec"
+          command: |
+            docker-compose exec web bundle exec rubocop -R --only RSpec
```

Comment:
I don't think you need to split these up.  Just running rubocop should cover the others.  It's been a while since I've set up rubocop, though, so i may be wrong.  I they're not running together, then we should find a way to get them to run together.

Reply:
It is running with Rubocop. This command would give the option to only run on Rubocop for Rspec.

Reply:
I mean that just running `bundle exec rubocop` should run all the cops (basic rubocop, rails, and rspec).  We don't want to split it up into 3 different instances.

Reply:
Ok, then we should remove the rubocop-rails and rubocop-rspec CI "run" instances and just do the regular rubocop one.  When we meet again we should go over how CI works and how these configs work to run certain commands.